.cirrus.yml

```diff
@@ -764,17 +764,23 @@ success_task:
 
     success_script: '/usr/local/bin/entrypoint.sh ./$SCRIPT_BASE/success.sh |& ${TIMESTAMP}'
 
+# Build the static binary
 static_build_task:
-    container:
-        image: quay.io/podman/nix-podman:1.0.0
-        cpu: 8
-        memory: 12
-    timeout_in: 20m
     depends_on:
         - "gating"
-    build_script:
-        - nix build -f nix
+    gce_instance:
+        image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+        cpu: 8
+        memory: 12
+        disk: 200
+    script:
+        - set -ex
+        - setenforce 0
+        - growpart /dev/sda 1 || true
+        - resize2fs /dev/sda1 || true
+        - yum -y install podman
+        - podman run --rm --privileged -ti -v ${PWD}:${PWD} -w ${PWD} nixos/nix nix --print-build-logs --option cores 8 --option max-jobs 8 build --file nix/
     binaries_artifacts:
-        path: "result-bin/bin/podman"
```

Comment:
nit: cirrus does not seem to publish the artifact correctly if I look at the job below: 
https://cirrus-ci.com/task/6505102414446592

Reply:
`-v /nix:/nix` is also required :-S

Reply:
ok finally get fixed ;-)

Reply:
see https://cirrus-ci.com/task/5346732219301888